test/support/ast_data.ex

```diff
@@ -159,11 +159,13 @@ defmodule Gradient.AstData do
 
           location =
             case {line, column} do
-              {l, c} when is_integer(l) and l >= 0 and is_integer(c) and c > 0 ->
+              {l, c}
+              when is_integer(l) and l >= 0 and
+                     is_integer(c) and c > 0 ->
                 {l, c}
 
-              other ->
-                raise "Could not normalize expression with anno: #{inspect(anno)} and acc #{inspect(acc)}, got: #{inspect(other)}, expression: #{inspect(expression)}"
+              _ ->
+                anno
```

Comment:
big assumption here that if didn't get a valid location, should return the original location as new location, makes any sense?